### PR TITLE
domRect getters

### PR DESCRIPTION
**Pull request by ccorcos**
@glennsl 

reference: https://developer.mozilla.org/en-US/docs/Web/API/DOMRect

### CODE REVIEW

**Comment by glennsl** [comment]
Looks good! Could you also add [the constructor](https://developer.mozilla.org/en-US/docs/Web/API/DOMRect/DOMRect) as `make` (similar to [this](https://github.com/BuckleTypes/reason-js/blob/master/src/dom/ranges/rangeRe.re#L3) but with more arguments), and a module alias in `DomRe.re`?

**Comment by ccorcos** [comment]
done

**Comment by glennsl** [comment]
Sorry for not clarifying this before. Could you add the module to [the list of modules here](https://github.com/BuckleTypes/reason-js/blob/master/src/dom/domRe.re)?

File: src/dom/geometry/domRectRe.re [hunk 1]
```diff
@@ -1,15 +1,19 @@
-external top : Dom.domRect => int = "" [@@bs.get];
+type t = Dom.domRect;
 
-external bottom : Dom.domRect => int = "" [@@bs.get];
+external make : int => int => int => int => t = "DOMRect" [@@bs.new]; /* experimental */
```
Comment (glennsl) [inline src/dom/geometry/domRectRe.re]: Could you also add labels to these arguments? With four arguments of the same type it's very easy to mistake them.

**Comment by ccorcos** [comment]
alrighty, let me know

**Comment by glennsl** [comment]
Thanks! I had to remove some text that had been randomly inserted, but otherwise it's good!

**Pull request merged**